lib/travis/build/appliances/deprecate_xcode_64.rb

```diff
@@ -0,0 +1,17 @@
+require 'travis/build/appliances/base'
+
+module Travis
+  module Build
+    module Appliances
+      class DeprecateXcode64 < Base
+        def apply
+          if config[:osx_image] == 'xcode6.4'
+            sh.echo ""
+            sh.echo "Running builds with Xcode 6.4 in Travis CI is deprecated and will be removed in January 2019.", ansi: :yellow
+            sh.echo "If Xcode 6.4 is critical to your builds, please contact our support team at support@travis-ci.com to discuss options.", ansi: :yellow
```

Comment:
should we also put a newline after this line too?

Reply:
Seems like there's already a blank line here in the log: https://staging.travis-ci.com/travis-infrastructure/Alamofire/jobs/219074